tests/unit/src/main/scala/tests/TestingClient.scala

```diff
@@ -349,6 +351,11 @@ class TestingClient(workspace: AbsolutePath, val buffers: Buffers)
           params.getMessage == ServerLivenessTestData.serverNotRespondingMessage
         ) {
           buildServerNotResponding
+        } else if (
+          params.getMessage() == FileOutOfScalaCliBspScope
+            .askToRegenerateConfigAndRestartBspMsg("SomeFile.scala")
```

Comment:
Could we maybe make sure this work for every file? Since if at any point a new test is added and SomeFile.scala is not used, this might be problematic. So something like:

```suggestion
          params.getMessage().startWith(FileOutOfScalaCliBspScope
            .askToRegenerateConfigAndRestartBspMsg(""))
```